nilearn/plotting/find_cuts.py

```diff
@@ -76,6 +76,8 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(),
                                                    80)
     mask = np.abs(my_map) > activation_threshold - 1.e-15
+    if mask.sum() == 0:
```

Comment:
What you are doing here is absolutely to me :). However, it seems that a comment would be welcome.

I addition, I would write the line above: 'if not np.any(mask)'.

Once this is in, and a test, I am :+1: for merge. And thanks, this is very useful!